blazor/getting-started/blazor-server-side-visual-studio.md

```diff
@@ -17,39 +17,41 @@ This article provides a step-by-step instructions for building Blazor Server App
 
 ## Create a new Blazor Server App in Visual Studio
 
-You can create **Blazor Server App** using Visual Studio in one of the following ways,
+You can create a **Blazor Server App** using Visual Studio via [Microsoft Templates](https://learn.microsoft.com/en-us/aspnet/core/blazor/tooling?view=aspnetcore-7.0) or the [Syncfusion Blazor Extension](https://blazor.syncfusion.com/documentation/visual-studio-integration/template-studio).
 
-* [Create a Project using Microsoft Templates](https://learn.microsoft.com/en-us/aspnet/core/blazor/tooling?pivots=windows&view=aspnetcore-7.0)
+## Install Syncfusion Blazor Packages in the App
```

Comment:
Single NuGet suggestion- Check the possibility to suggest individual nuget

Reply:
Resolved the mentioned feedbacks.